postgresql/DockerScripts/3dcitydb-initdb.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+# 3DCityDB setup --------------------------------------------------------------
+
+# Print commands and their arguments as they are executed
+set -e;
+
+# psql should stop on error
+psql=( psql -v ON_ERROR_STOP=1 )
+
+# SRID ------------------------------------------------------------------------
+regex_numeric='^[0-9]+$'
+if [ -z ${SRID+x} ]; then
+  # No SRID set -> give instructions on how to create a DB and do nothing
+  echo
+  echo "SRID is not set. No 3DCityDB instance will be created in database '$POSTGRES_DB'."
+  exit
+else
+  # SRID given, check if valid
+  if [[ ! $SRID =~ $regex_numeric ]] || [ $SRID -le 0 ]; then
+        echo
+        echo 'Illegal input! Enter a positive integer for the SRID.'
+  fi
+fi
+
+# GMLSRSNAME  -----------------------------------------------------------------
+if [ -z ${GMLSRSNAME+x} ]; then
+  # GMLSRSNAME unset, set default GMLSRSNAME using HEIGHT_EPSG if set
+  # HEIGHT EPSG ---------------------------------------------------------------
+  if [ -z ${HEIGHT_EPSG+x} ]; then
+    # No HEIGHT_EPSG given
+    GMLSRSNAME="urn:ogc:def:crs:EPSG::$SRID"
+  else
+    if [ $HEIGHT_EPSG -gt 0 ]; then
+      GMLSRSNAME="urn:ogc:def:crs,crs:EPSG::$SRID,crs:EPSG::$HEIGHT_EPSG"
+    else
+      GMLSRSNAME="urn:ogc:def:crs:EPSG::$SRID"
+    fi
+  fi
+else
+  if [ ! -z ${HEIGHT_EPSG+x} ]; then
+    # GMLSRSNAME is set, HEIGHT_EPSG is ignored
+    echo
+    echo "!!! WARNING: GMLSRSNAME is set. HEIGHT_EPSG will be ignored."
+  fi
+fi
+
+# Add PostGIS raster extension ------------------------------------------------
+# Get major version from POSTGIS_VERSION, POSTGIS_MAJOR may return string
+postgis_major=$( echo $POSTGIS_VERSION | cut -f1 -d '.' )
+if [ $postgis_major -gt 2 ]; then
+  echo
+  echo "Create PostGIS raster extensions in database '$POSTGRES_DB' ..."
+
+  "${psql[@]}" -d "$POSTGRES_DB" -c "CREATE EXTENSION IF NOT EXISTS postgis_raster;"
+
+  echo "Create PostGIS raster extensions in database '$POSTGRES_DB' ...done!"
+fi
+
+# Add PostGIS SFCGAL extension ------------------------------------------------
+if [ ! -z ${POSTGIS_SFCGAL+x} ]; then
+  # SFCGAL is currently not available in the post/postgis:alpine images.
+  # Test for Alpine Linux and warn if enabled and alpine image variant.
+
+  grep -iE "alpine" /etc/os-release > /dev/null
+  if [ "$?" = 0 ]; then
+    echo
+    echo "!!! WARNING: POSTGIS_SFCGAL was set, but SFCGAL is not available on Alpine Linux image variants."
+    echo "!!! Please use the Debian based image variants if you require SFCGAL support."
+    SFCGAL=true
```

Comment:
@BWibo thanks. I have only one question after my final review.  From my understanding, should this line be `SFCGAL=false`?

Reply:
Yes, thx for spotting this, I fixed it.